lib/libp2p.go

```diff
@@ -4,10 +4,11 @@ import (
 	"github.com/libp2p/go-libp2p"
 	mplex "github.com/libp2p/go-libp2p-mplex"
 	"github.com/libp2p/go-libp2p/core/host"
+	libp2pwebrtc "github.com/libp2p/go-libp2p/p2p/transport/webrtc"
 )
 
 func libp2pHost() (host.Host, error) {
-	h, err := libp2p.New(libp2p.DefaultMuxers, libp2p.Muxer("/mplex/6.7.0", mplex.DefaultTransport), libp2p.EnableHolePunching())
+	h, err := libp2p.New(libp2p.DefaultTransports, libp2p.Transport(libp2pwebrtc.New), libp2p.DefaultMuxers, libp2p.Muxer("/mplex/6.7.0", mplex.DefaultTransport), libp2p.EnableHolePunching())
```

Comment:
@MarcoPolo any downsides here? This will disable listen addresses, but that seems fine (even though it'll prevent dialbacks if the node is publicly reachable). Perhaps we should even have been doing that explicitly or enabling the user to pass a listen address if they want dialbacks to work.

Reply:
This might break hole-punching too.

Reply:
Why would this break hole-punching?

Isn't this a problem then now that we have [webrtc enabled by default](https://github.com/libp2p/go-libp2p/blob/61507e54767e7a0a072122ddbbee63531c7e30b1/defaults.go#L51)?